tests/installation/registration/register_via_scc.pm

```diff
@@ -7,19 +7,21 @@
 # notice and this notice are preserved.  This file is offered as-is,
 # without any warranty.
 
-# Summary: Register the system with scc in the installer with registration code.
+# Summary: Register the system via SCC in the installer with registration
+# code, optional email and enabling update repositories.
 #
 # Maintainer: QA SLE YaST team <qa-sle-yast@suse.de>
 
 use base 'y2_installbase';
 use strict;
 use warnings;
-use testapi 'get_var';
```

Comment:
I use to specify just the functions I need in this case, I suppose it gives a tiny bit of memory optimization by loading in memory only what we need. Though this is just an assumption, I could not find a doc to confirm it, and I guess we would barely notice the difference on modern hardware anyway.

Reply:
It is so basic library, that once you use some function you might need some other when you need to fix this test, so it is more comfortable in this case to have the general one for maintenance and debugging.